js/src/tab.js

```diff
@@ -229,6 +217,48 @@ class Tab {
     })
   }
 
+  static _dataApiKeydownHandler(event) {
+    const tablist = SelectorEngine.closest(event.target, Selector.TABLIST)
+    let tablistorientation = tablist.getAttribute('aria-orientation')
```

Comment:
use `const` here and maybe a bit of camel case, for example: `const tabListOrientation`

Reply:
but I'm potentially changing the value of `tablistorientation` in the next bit, so not a constant?

Reply:
in that case `let` if fine 👍